src/queries/label-text.js

```diff
@@ -38,15 +48,7 @@ function queryAllLabelsByText(
 
 function getTextContent(node) {
   const TEXT_NODE = 3
```

Comment:
I know you moved `labelledNodeNames` because it is used by `queryAllByLabelText` too but maybe you could move outside `getTextContent` the declaration of `TEXT_NODE` too.

Anyway this PR looks legit to me! 👍 😄

Reply:
hahaha yes